### ISSUE TITLE
[Accessibility]A11y_VSCode_ColorTheme_ColorContrast:Color contrast ratio is less than 4.5:1 for the Red colored content when not focused.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [chrmarti/testissues#22785]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Navigate to Settings-->Select color theme -->Select solarized light
2)Check color contrast ratio for the Explorer content on the screen.

**Actual:**
Color contrast ratio is less than 4.5:1 for the Red colored content when not focused.
**Similar issue is observed in following themes:**

Dark visual studio
Dark Default dark
Kimbie dark
Monokai
Monokai Dimmed
Red
Solarized dark
Tomorrow night blue

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content on a screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference:**
![accessibility a11y_vscode_colortheme explorer _colorcontrast notfocused](https://user-images.githubusercontent.com/39204909/41973383-746b3626-7a32-11e8-825b-35daec1ec7a9.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
Unfortunately bug was logged in this path.Hence closing this bug.
